src/main/java/io/nats/client/KeyValue.java

```diff
@@ -196,7 +193,8 @@
     List<KeyValueEntry> history(String key) throws IOException, JetStreamApiException, InterruptedException;
 
     /**
-     * Remove history from all keys that currently are deleted or purged.
+     * Remove history from all keys that currently are deleted or purged
+     * with using a default KeyValuePurgeOptions
      * THIS IS A BETA FEATURE AND SUBJECT TO CHANGE
```

Comment:
Can remove this if this makes the KV API GA.  Alternatively, that could be another PR - up to you.